grafana-ui: failed tests in 3d party plugins

After fixing #20921 I get another error when run tests. Tests are failed on files where `@grafana/ui` imported, like 
```ts
import { withTheme, Input, EventsWithValidation, ValidationEvents, Themeable } from '@grafana/ui';
```
Since I'm not building plugin with `@grafana/ui` (grafana libs are excluded in my webpack config:
```js
externals: [
    // remove the line below if you don't want to use builtin versions
    'jquery', 'lodash', 'moment', 'angular', 'emotion',
    'react', 'react-dom', '@grafana/ui', '@grafana/data', '@grafana/runtime',
```
), I get following error only with Jest tests:
```
 FAIL  src/datasource-zabbix/specs/datasource.spec.js
  ● Test suite failed to run

    Cannot find module '@grafana/data/src/datetime/datemath' from 'index.development.js'

    However, Jest was able to find:
    	'../module.ts'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['ts', 'tsx', 'js', 'jsx', 'json'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:13:16)
```

I've found a root cause - import in one of time picker components looks strange:
https://github.com/grafana/grafana/blob/43dbbe51f0e05e87e684700c28034d16014d917f/packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimeRangeForm.tsx#L7

Fix is pretty simple - just import `dateMath` and use `isMathString` like `dateMath.isMathString()`:
```ts
import { TIME_FORMAT, TimeZone, isDateTime, TimeRange, DateTime, dateMath } from '@grafana/data';

function isValid(value: string, roundup?: boolean, timeZone?: TimeZone): boolean {
  if (dateMath.isMathString(value)) {
    return isValidTimeString(value);
  }

  const parsed = stringToDateTimeType(value, roundup, timeZone);
  return parsed.isValid();
}

--- a/packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimeRangeForm.tsx
+++ b/packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimeRangeForm.tsx
@@ -1,10 +1,9 @@
 import React, { FormEvent, useState, useCallback } from 'react';
-import { TIME_FORMAT, TimeZone, isDateTime, TimeRange, DateTime } from '@grafana/data';
+import { TIME_FORMAT, TimeZone, isDateTime, TimeRange, DateTime, dateMath } from '@grafana/data';
 import { stringToDateTimeType, isValidTimeString } from '../time';
 import { mapStringsToTimeRange } from './mapper';
 import { TimePickerCalendar } from './TimePickerCalendar';
 import Forms from '../../Forms';
-import { isMathString } from '@grafana/data/src/datetime/datemath';
 
 interface Props {
   isFullscreen: boolean;
@@ -116,7 +115,7 @@ function valueAsString(value: DateTime | string): string {
 }
 
 function isValid(value: string, roundup?: boolean, timeZone?: TimeZone): boolean {
-  if (isMathString(value)) {
+  if (dateMath.isMathString(value)) {
     return isValidTimeString(value);
   }
 

